docs/src/heat_equation.md

```diff
@@ -0,0 +1,217 @@
+# [Tutorial: Solution of the heat equation](@id tutorial-heat-equation)
```

Comment:
This extends the linear advection tutorial with the use of tridiagonal matrices. From my point of view we don't need the linear advection tutorial any longer.

We could also use Dirichelt boundary conditions here and use this as a tutorial for a `PDSProblem`.

Reply:
I think it's fine to keep the linear advection tutorial as well since we have periodic BCs there and a different setup in general (FD instead of FV).